godotproject/Tiles/starryNight/starryNight.gd

```diff
@@ -0,0 +1,3 @@
+extends Node2D
+func _ready():
+	$AnimationPlayer.play("comet")
```

Comment:
There's no way to automatically play an animation as far as I can tell >:(

Reply:
👿